src/components/Tutorial/Assessments/Quiz.vue

```diff
@@ -12,7 +12,8 @@
   <div class="quiz">
     <ContentNode class="title" :content="title" />
     <ContentNode class="question-content" v-if="content" :content="content" />
-    <div class="choices">
+    <fieldset class="choices">
+      <legend class="visuallyhidden">{{ $t('tutorials.assessment.legend') }}</legend>
```

Comment:
Should we worry this is copyable?

Reply:
In my opinion, this UI doesn't have text that it's meant to be copiable. Also, I would prefer to use this HTML standard than a workaround to avoid copyable text... but any alternative idea is welcome!